src/views/transactions/components/transaction-form/transaction-form.view.jsx

```diff
@@ -260,7 +263,7 @@ function TransactionForm({
         const accountChecks = [
           getAccounts(receiver, [account.token.id]),
           ...(!isHermezBjjAddress(receiver)
-            ? [getCreateAccountAuthorization(receiver).catch(() => undefined)]
+            ? [getCreateAccountAuthorization(receiver).catch(() => ({}))]
```

Comment:
This is breaking the create authorization check in the form! If we return something different than `undefined` or `null` the check is going to break and will allow to send the tx without having the create account auth signature.

Reply:
I think that we need to review this logic deeper

Reply:
Yep, we will do it when we refactor this component

Reply:
I [explicitly asked](https://0xpolygon.slack.com/archives/C02FS3PHEGH/p1635513098067600) about this and [you agreed](https://0xpolygon.slack.com/archives/C02FS3PHEGH/p1635518824074300) to leave the `{}`.

Then it needs to be `undefined` because we're checking for a falsy value. Ok!

Reply:
Done. Added a comment to clarify this.

Reply:
Yep, I thought that you were talking about a different thing, my fault, sorry!